apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ThrowableExpectationSamples.kt

```diff
@@ -26,37 +26,36 @@ class ThrowableExpectationSamples {
 
     @Test
     fun message() {
-        expect(RuntimeException("abc")).message {
+        expect(RuntimeException("abc")).message { // subject inside this block is of type String (actually "abc")
             toContain("a")
         }
 
         fails {
-            expect(RuntimeException("abc")).message {
+            expect(RuntimeException("abc")).message { // subject inside this block is of type String (actually "abc")
                 toContain("d")
             }
         }
     }
 
     @Test
     fun causeFeature() {
-        expect(IllegalStateException(IndexOutOfBoundsException())).cause<Throwable>()
-            .toBeAnInstanceOf<IndexOutOfBoundsException>()
+        expect(IllegalStateException(IndexOutOfBoundsException("abc"))).cause<IndexOutOfBoundsException>()
+            .messageToContain("b")
 
         fails {
-            expect(IllegalStateException(IndexOutOfBoundsException())).cause<Throwable>()
-                .toBeAnInstanceOf<IllegalStateException>()
+            expect(IllegalStateException(IndexOutOfBoundsException("abc"))).cause<IllegalStateException>()
         }
     }
 
     @Test
     fun cause() {
-        expect(IllegalStateException(IndexOutOfBoundsException())).cause<Throwable> {
-            toBeAnInstanceOf<IndexOutOfBoundsException>()
+        expect(IllegalStateException(IndexOutOfBoundsException("abc"))).cause<IndexOutOfBoundsException> { // subject inside this block is of type Throwable (actually IndexOutOfBoundsException)
```

Comment:
The subject is of type IndexOutOfBoundsException that's also the reason the previous `toBeInstanceOf` is unnecessary.
`cause` is a special case, the subject also remains IndexOutOfBoundsException after the block because it is a type check. if `cause` passes then the subject has to be `IndexOutOfBoundsException` and stays this type

```suggestion
        expect(IllegalStateException(IndexOutOfBoundsException("abc"))).cause<IndexOutOfBoundsException> { // subject is now of type IndexOutOfBoundsException
```